src/main/kotlin/ua/marchenko/artauction/common/annotation/scheduled/cron/enums/Day.kt

```diff
@@ -0,0 +1,3 @@
+package ua.marchenko.artauction.common.annotation.scheduled.cron.enums
+
+enum class Day { MONDAY, TUESDAY, WEDNESDAY, THURSDAY, FRIDAY, SATURDAY, SUNDAY, ALL }
```

Comment:
I'm ready to bet that same enum exists somewhere in JDK

Reply:
I have found `DayOfWeek`, but there is no values for ALL (like every day)

Reply:
you can achieve the same result by having sealed interface with two implementations: Daily and Weekly instead of one DayTimeDetails

Reply:
I don't understand the idea. So would annotation have the parameter of sealed interface and inside of BPP I would set delay based on type of Weekly or Daily?